go/src/meguca/server/router.go

```diff
@@ -43,8 +43,9 @@ var (
 
 	isTest bool
 
-	errNoYoutubeVideo = common.StatusError{errors.New("no YouTube video found"),
-		404}
+	errYouTubeLive = common.StatusError{errors.New("YouTube video is a livestream"), 501}
```

Comment:
>501 
This will get logged as a server error. Change to a 415.

These should also be converted to error functions, that take a youtube ID and output an error that includes the ID in the message.